### ISSUE TITLE
incompatibility of file_picker && parse_server_sdk_flutter

### ISSUE THREAD

**Issue by catalunha** [parse-community/Parse-SDK-Flutter#944]
### New Issue Checklist
- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-SDK-Flutter/security/policy).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/main/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/Parse-SDK-Flutter/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest version of [Parse Server](https://github.com/parse-community/parse-server/releases) and the [Parse Flutter SDK](https://github.com/parse-community/Parse-SDK-Flutter/releases). 

### Issue Description
incompatibility of file_picker && parse_server_sdk_flutter
pubspec.yaml
```
dependencies:
  # Back4app
  parse_server_sdk_flutter: ^5.1.0
  # image/file
  file_picker: ^5.2.11
```

output
```
[fluxus3] flutter pub get
Resolving dependencies...
Because no versions of file_picker match >5.3.2 <6.0.0 and file_picker 5.3.2 depends on win32 ^5.0.2, file_picker ^5.3.2 requires win32 ^5.0.2.
And because parse_server_sdk_flutter >=4.0.0 depends on package_info_plus ^3.0.3 which depends on win32 >=2.7.0 <5.0.0, file_picker ^5.3.2 is incompatible with parse_server_sdk_flutter >=4.0.0.
So, because fluxus3 depends on both parse_server_sdk_flutter ^5.1.0 and file_picker ^5.3.2, version solving failed.
exit code 1
```

### Steps to reproduce
Import packegs to generate problem with dependencies

### Actual Outcome
version solving failed

### Expected Outcome
version solving success

### Environment

Parse Flutter SDK
- SDK version: `5.1.0`
- Flutter version: `3.10.5`
- Dart version: `3.0.5`
- Operating system version: `POP`

Server
- Parse Server version: `4.10.4`

**Comment by catalunha** [comment]
Thanks to the entire update team. Sorry to post an issue without acknowledgment at the beginning. But it was running on time.

Parsing Flutter SDK 5.0.1 works.

**Comment by kirkmorrow** [comment]
Latest version works with the latest version of file_picker.  Seems like this issue can be closed.

**Comment by mtrezza** [comment]
Thanks for checking @kirkmorrow, closing.
